Improve doc: update optional installation dependencies (#13)
autogenhub/autogen: website/docs/installation/Optional-Dependencies.md

# Optional Dependencies

## Different LLMs

AutoGen installs OpenAI package by default. To use LLMs by other providers, you can install the following packages:

```bash
pip install pyautogen[gemini,anthropic,mistral,together,groq,cohere]
```

> Check out the [notebook](https://github.com/autogen-ai/autogen/blob/main/notebook/autogen_uniformed_api_calling.ipynb) and

Reviewer: ```suggestion
Check out the [notebook](/docs/notebooks/autogen_uniformed_api_calling.ipynb) and
```

Reviewer: Anything comment before resolving it?

> [blogpost](../../blog/2024/06/24/AltModels-Classes) for more details.

Reviewer: ```suggestion
[blogpost](/blog/2024/06/24/AltModels-Classes) for more details.
```

Author: Sorry I put the comment in the resolved thread so it is not shown.

From the code above, the compile has errors and the web page is not shown, that is why I modified it.
For code below, I am changing the notebook link back to URL since it is not compiled in the website.

Author: My mistake, the reference should work. @sonichi I have fixed the broken urls. Can you take a look now?

pip install "pyautogen[lmm]"
```

> Example notebook: [LLaVA Agent](https://github.com/autogen-ai/autogen/blob/main/notebook/agentchat_lmm_llava.ipynb)

Reviewer: ```suggestion
Example notebook: [LLaVA Agent](/docs/notebooks/agentchat_lmm_llava.ipynb)
```
